apps/modernization-api/src/main/java/gov/cdc/nbs/patient/PatientService.java

```diff
@@ -141,7 +149,24 @@ public Page<Person> findPatientsByFilter(PatientFilter filter, GraphQLPage page)
         builder.must(QueryBuilders.matchQuery(ElasticsearchPerson.CD_FIELD, "PAT"));
 
         if (filter.getId() != null) {
-            builder.must(QueryBuilders.matchQuery(ElasticsearchPerson.LOCAL_ID, filter.getId()));
+            if (Character.isDigit(filter.getId().charAt(0))) {
+                try {
+                    long shortId = Long.parseLong(filter.getId());
+                    PatientIdentifierSettings settings = new PatientIdentifierSettings(
+                        patientIdPrefix,
+                        patientIdSeed,
+                        patientIdSuffix
+                    );
+                    PatientLocalIdentifierResolver resolver = new PatientLocalIdentifierResolver(settings);
```

Comment:
Should be able to inject the `PatientLocalIdentifierResolver` directly instead of having to create it each time.